x-pack/plugins/infra/public/alerting/metric_threshold/components/expression.test.tsx

```diff
@@ -55,6 +55,7 @@ describe('Expression', () => {
           currentOptions,
         }}
         dataViews={dataViewMock}
+        onChangeMetaData={() => {}}
```

Comment:
This event doesn't seem to be used in the Expressions component. [Should we omit ](https://github.com/elastic/kibana/blob/main/x-pack/plugins/infra/public/alerting/metric_threshold/components/expression.tsx#L43) `onChangeMetaData` instead, to avoid confusing the expectations for this component?